src/js/lib/DOMTranslation.ts

```diff
@@ -1091,7 +1097,7 @@ class DOMTranslation {
     firstLevel: boolean = false
   ) {
     try {
-      if (element.nodeType === Node.TEXT_NODE && element.textContent.trim().length === 0) {
+      if (element.nodeType === Node.TEXT_NODE && element.textContent.trim().length === 0 && !this.pluginOptions.translation.translateWholePage) {
```

Comment:
Šis bija vajadzīgs, lai neapstrādātu elementus bez teksta satura? Ja jā, kāpēc vajadzīgs tas papildus nosacījums?

Reply:
tiks noņemts, nav vajadzīgs